src/containers/LoginPage/statiticsTable.js

```diff
@@ -46,7 +46,11 @@ export default class StatiticsTable extends Component {
       })
       .then((responseJson) =>
       {
-        this.setState({ rows: responseJson.data });
+          const data = responseJson.data.filter(d => {
+              return (d['numberWantToLearn'] > 0 || d['numberWantToTeach'] > 0);
```

Comment:
I think it's better to use the object notation, `d.numberWantToLearn`

Reply:
@janakavishwajith  i have made the update